Envoy: Reflect cilium log level to Envoy.

jrajahalme opened the pull request:
Simplify interaction with Envoy by requiring that debug version uses
the same binary name.

Use the Envoy admin interface to set the log level at runtime whenever
cilium log level is changed.

Disable Cilium debug output for the duration of the stress tests. This
sets the Cilium internal log level to "info", which is reflected to
the Envoy process using the admin interface. This drastically reduces
the logging output of the Envoy process, as on the debug level a
stress test can fill a disk with logs.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

raybejjani commented on pkg/envoy/envoy.go:
```diff
@@ -101,24 +160,16 @@ func StartEnvoy(debug bool, adminPort int, stateDir, logDir string, baseID uint6
-				logLevel = "debug"
+			logLevel := envoyLevelMap[log.Level]
+			if logLevel == "" {
+				logLevel = "info"
```
maybe set the value via a map lookup, so that the map is the only source of values passed to envoy?

jrajahalme replied:
Good point! Done for the upcoming build with a rebase to master as well.

jrajahalme committed:
Envoy: Reflect cilium log level to Envoy.

Simplify interaction with Envoy by requiring that debug version uses
the same binary name.

Use the Envoy admin interface to set the log level at runtime whenever
cilium log level is changed.

Disable Cilium debug output for the duration of the stress tests. This
sets the Cilium internal log level to "info", which is reflected to
the Envoy process using the admin interface. This drastically reduces
the logging output of the Envoy process, as on the debug level a
stress test can fill a disk with logs.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>
